src/components/CreateCoursePage/CreateCourseForm.test.jsx

```diff
@@ -25,13 +25,17 @@ const courseRunOptionsReturn = {
 const parseOptionsReturn = [{ label: 'Instructor-paced', value: 'instructor_paced' },
   { label: 'Self-paced', value: 'self_paced' }];
 
+Date.now = jest.fn(() => new Date(Date.UTC(2001, 0, 1)).valueOf());
+
 describe('CreateCourseForm', () => {
   const initialValues = {
     org: 'edx',
     title: 'Hello',
     number: 'edx101',
     enrollmentTrack: 'verified',
     price: 100.00,
+    start: '2001-01-01T12:00:00Z',
```

Comment:
Are these necessary? They don't seem to show up in the snapshot anywhere outside of the initialValues and currentFormValues so just double checking the reason for adding them. Also commenting because you make the change here, but not in the CreateCourseRunForm.test.jsx file.

Reply:
Good catch, these were unnecessary.